helm/akhq/values.yaml

```diff
@@ -42,11 +41,18 @@ secrets: |
             url: "http://connect:8083"
             basic-auth-username: basic-auth-user
             basic-auth-password: basic-auth-pass
-
 # Any extra volumes to define for the pod (like keystore/truststore)
 extraVolumes: []
 # Any extra volume mounts to define for the akhq container
 extraVolumeMounts: []
+# Add your own init container or uncomment and modify the example.
+initContainers: 
```

Comment:
Can you comment the example please ? it will start for every people ;=)

Reply:
fail ! it's done ^^